lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -1035,49 +1039,44 @@ static LogicalResult buildAllocOp(ComponentLoweringState &componentState,
             bit_cast<uint64_t>(fltAttr.getValueAsDouble());
       } else {
         auto intAttr = dyn_cast<mlir::IntegerAttr>(attr);
-        int64_t value = intAttr.getInt();
```

Comment:
because `getInt()` only works for signless or index type. So switching to use getting an `APInt value` to handle signed and signless uniformly